src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA `spring_introduce` DEFAULT CHARACTER SET utf8 ;
```

Comment:
we don't need this file, since we will work with hibernate and spring

Reply:
Fixed.